Makefile

```diff
@@ -51,6 +52,21 @@ MD_FILES := $(shell \
 		-name '*.md' \
 	| sed 's/ /\\ /g' \
 )
+UI_FILES = $(shell \
```

Comment:
Do we need an explicit list of files? How would we maintain it? From the name: “UI” I don’t immediately understand which files should I include.